docs/docs/wikitext103.md

```diff
@@ -0,0 +1,205 @@
+# WikiText-103
+
+![Banner](img/banner.png)
+[//]: # (TODO Change Me) 
+
+You can download WikiText-103 dataset [here](https://blog.einstein.ai/the-wikitext-long-term-dependency-language-modeling-dataset/), there is no official leaderboard.
```

Comment:
link to the sotabench leaderboard (as with the other docs for each task)

Reply:
>Unless absolutely necessary, I would not try to deviate too far from the API style of the other benchmarks - e.g. using new methods like eval, not using .save() etc. We might want to change these in future but it adds more complexity to have a completely independent approach to doing benchmarks

Ross that is just a proposal, both  API works, the eval is added to the base  class so it is available everywhere where the base class is used. I can remove that from the docs and I can remove the code. I understand you don't like the approach with  generators, i suppose you think it makes it more complicated for arbitrary ml person to follow, although this approach come with prons as well:
- you can't forget about  `save` or `cache_exists`
- it gives us more control over the calls to the model, it should be easier to measure the memory usage before first batch.
- it will be easier to catch other statistics, and to catch errors during execution.

But as I said above it is just a proposal I wanted you to see how it would look like, you find it complex let's have it removed.

>  speed calculation on the platform before the launch (so we can see speed vs perplexity of the different models).
I though it was added to the base class, good catch let me make sure it is working.

Reply:
Sure - no problem. I understand the arguments. I think I would opt for between-benchmark-consistency in this case, just because it's additional overhead if there's two ways of doing things (even if one takes care of more things automatically!). We can discuss how we can improve the API in general from next week onwards; might be that we adopt something closer to this proposal!